src/include/reconcile.h

```diff
@@ -309,7 +309,8 @@ typedef struct {
 
     WT_TIME_WINDOW tw;
 
-    bool upd_saved; /* An element on the row's update chain was saved */
+    bool upd_saved;           /* An element on the row's update chain was saved */
+    bool ooo_or_mm_tombstone; /* Tombstone is out-of-order or mixed-mode */
```

Comment:
Can we just call it out of order tombstone? Mixed mode is a type of out of order.

Reply:
Done.